src/defs.hpp.in

```diff
@@ -23,10 +24,17 @@
 // Riemann solver
 #define RIEMANN_SOLVER "@RSOLVER@"
 
-// configure.py dict(definitions) Boolean values:
-// Equation of state
+// equation of state
 #define EQUATION_OF_STATE "@EQUATION_OF_STATE@"
 
+// ChemNetwork class header file included in species.hpp
+#define CHEMNETWORK_HEADER "@CHEMNETWORK_HEADER@"
+
+// ChemRadiation integrator method, used in constructing chemradiation tasklist
+#define CHEMRADIATION_INTEGRATOR "@CHEMRADIATION_INTEGRATOR@"
```

Comment:
am I correct in assuming that these two string macros are always identical, save `.hpp` file suffix on the `CHEMNETWORK_HEADER`. Can we reduce these to 1x macro?

Reply:
No these two are different. `CHEMNETWORK_HEADER` is for the chemistry network (such as gow17, H2 etc.), `CHEMRADIATION_INTEGRATOR` is for the radiation scheme (const or six_ray).

Reply:
ah of course, that is what I get for reviewing late at night. Got this confused with the Python definitions:
```
    makefile_options['CHEMRADIATION_FILE'] = args['chem_radiation']+'.cpp'
    definitions['CHEMRADIATION_INTEGRATOR'] = args['chem_radiation']
```